sensor_msgs/Image image_front
sensor_msgs/Image image_left_back
sensor_msgs/Image image_right_back
sensor_msgs/PointCloud2 cloud
---
sensor_msgs/PointCloud2 color_cloud
